llpc/lower/llpcSpirvLowerGlobal.h

```diff
@@ -54,11 +54,12 @@ class SpirvLowerGlobal : public SpirvLower, public llvm::PassInfoMixin<SpirvLowe
 
   void handleLoadInst();
   void handleLoadInstGlobal(LoadInst &loadInst, const unsigned addrSpace);
-  void handleLoadInstGEP(GetElementPtrInst *const getElemPtr, LoadInst &loadInst, const unsigned addrSpace);
+  void handleLoadInstGEP(SmallVectorImpl<Value *> &indexOperands, LoadInst &loadInst, const unsigned addrSpace,
+                         GlobalVariable *inOut);
```

Comment:
Please keep related operands close together. I would suggest:
```
void handleLoadInstGEP(GlobalVariable *inOut, SmallVectorImpl<Value *> &indexOperands, LoadInst &loadInst);
```
(addrSpace is implied by the GlobalVariable, isn't it?)